libs/remix-ui/workspace/src/lib/reducers/workspace.ts

```diff
@@ -673,9 +673,7 @@ const fetchDirectoryContent = (state: BrowserState, payload: { fileTree, path: s
     }
   } else {
     if (payload.path === state.mode || payload.path === '/') {
-      let files = normalize(payload.fileTree, payload.path, payload.type)
-
-      files = _.merge(files, state[state.mode].files[state.mode])
+      const files = normalize(payload.fileTree, payload.path, payload.type)
```

Comment:
We merge because this creates a bug in remix-desktop.

Reply:
@Aniket-Engg  if the publish version is the one you want  … I tested it on Desktop and @ioedeveloper is correct. if the tree is expanded and you add something in root the tree will collapse.